src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import classNames from 'classnames';
+import { TypeFilterin } from '../../types/FilterTypes';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  setTypeOfFiltering: (x:TypeFilterin) => void;
+  typeOfFiltering: TypeFilterin;
+};
+
+export const Footer:React.FC<Props> = ({
+  todos,
+  setTypeOfFiltering,
+  typeOfFiltering,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${todos.filter(todo => todo.completed === false).length} items left`}
```

Comment:
Use logical operators. And move the filtered value to a variable. Don't calculate it inside jsx 
```suggestion
        {`${todos.filter(todo => !todo.completed).length} items left`}
```